database/factories/UserFactory.php

```diff
@@ -10,33 +10,34 @@
  */
 class UserFactory extends Factory
 {
-    /**
-     * Define the model's default state.
-     *
-     * @return array<string, mixed>
-     */
-    public function definition()
-    {
-        return [
-            // 'name' => fake()->name(),
-            // 'email' => fake()->safeEmail(),
-            // 'email_verified_at' => now(),
-            // 'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
-            // 'remember_token' => Str::random(10),
-        ];
-    }
+	/**
+	 * Define the model's default state.
+	 *
+	 * @return array<string, mixed>
+	 */
+	public function definition()
+	{
+		return [
+			'firstname' => fake()->name(),
+			'lastname' => fake()->lastName(),
+			'email' => fake()->safeEmail(),
+			// 'email_verified_at' => now(),
```

Comment:
remove if not use

Reply:
You mean line 24 right? If so, it is removed.